dotnet/src/SemanticKernel/Orchestration/SKContextExtensions.cs

```diff
@@ -15,13 +15,20 @@ internal static class SKContextExtensions
     /// <param name="skillName">The skill name</param>
     /// <param name="functionName">The function name</param>
     /// <param name="registeredFunction">The registered function, if found</param>
-    internal static bool IsFunctionRegistered(this SKContext context, string skillName, string functionName, [NotNullWhen(true)] out ISKFunction? registeredFunction)
+    internal static bool IsFunctionRegistered(this SKContext context, 
```

Comment:
This returns an object in the out param and bool=true if found...
Can we rename to TryGetFunction, to align with ,net and similar calls throughout the Kernel?

Consider also the use of "registered" in the args and method comments... if we're removing the "Register*" naming in favor of "Import", it might be a good idea to replace it here too for consistency.

Reply:
Or `Added` 
We have three different terms `Import` `Register` `Add`, would be good to stick to one.

Reply:
`Register` can go away probably. `Add` is for implementation, `Import` is for user space.